### DISCUSSION TITLE
Linux Interface rewrite status

### DISCUSSION THREAD

**Discussion by Paolo86cripple** [86Box/86Box#1715] [General]
Hi there, is there some kind of update about the status of the 86box linux port and the interface rewrite?

**Comment by dhrdlicka** [comment]
You might want to join the Discord, there is a thread dedicated to the UI rewrite.

**Reply by Paolo86cripple** [reply]
Will head down there ASAP! Thanks!

**Comment by jriwanek** [comment]
It's released: https://github.com/86Box/86Box/releases/tag/v3.2.1

**Comment by jriwanek** [comment]
Closing as discussion resolved
